src/main/java/org/jenkinsci/test/acceptance/po/Control.java

```diff
@@ -112,6 +112,15 @@ public void setAtOnce(String text){
         ((JavascriptExecutor)driver).executeScript("arguments[0].value = arguments[1];", e, text);
     }
 
+    /**
+     * Returns the value of the input field.
+     *
+     * @return the value of the input field.
+     */
+    public String get() {
+        WebElement e = resolve();
+        return ((JavascriptExecutor)driver).executeScript("return arguments[0].value;", e).toString();
```

Comment:
`WebElement#getAttribute("value")` works fine. Should I remove the method `Control#get()` and do it in `JobDslBuildStep`?
In inputs like textarea and select the method works, but in inputs like checkbox or radio I get nothing useful.